opensearch-operator/pkg/reconcilers/securityconfig.go

```diff
@@ -204,3 +175,30 @@ func (r *SecurityconfigReconciler) DeleteResources() (ctrl.Result, error) {
 	result := reconciler.CombinedResult{}
 	return result.Result, result.Err
 }
+func (r *SecurityconfigReconciler) SecurityconfigSubpaths(instance *opsterv1.OpenSearchCluster, secret *corev1.Secret) error {
```

Comment:
Just a nit; if this function isn't getting called outside of the package it should be an unexported function (lowercase starting letter).

Reply:
Fixed in my latest commit @dbason.